hack/community/readme.md

```diff
@@ -1,45 +1,24 @@
 # Cut a new community release
-The generate.sh script automates the community release process by copying over 
-the latest WMCO manifests to a new community-WMCO folder while also replacing 
-necessary fields such as the name, description, and image. It also generates a 
-signed commit ready to be pushed to community-operators-prod repo. 
+The generate.sh script is triggered to set up for the aws-e2e-operator-test 
+in all community-4.10+ branch PRs. The script copies over the WMCO community 
```

Comment:
`community-4.10+ branch PRs`  which branches this applies to does not need to be mentioned, the lack of the script + doc in community-4.9 and below makes that clear.